NEWS

```diff
@@ -1,6 +1,19 @@
-libva NEWS -- summary of user visible changes.  2019-12-25
+libva NEWS -- summary of user visible changes.  2010-03-26
```

Comment:
```suggestion
libva NEWS -- summary of user visible changes.  2020-03-26
```
?

And 2019->2020 below.

Reply:
Also to be consistent, the format of the date below should be 26.Mar.2020 not `March`

Reply:
accept